distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFtp.java

```diff
@@ -88,6 +91,7 @@ public void afterExam() throws Exception {
         try {
             // Turn off feature to not interfere with other tests
             getServiceManager().stopFeature(true, FTP_ENDPOINT_FEATURE);
+            //TODO Reset system.properties port after finishes tests?
```

Comment:
Each test class will get a new distribution and have it's own copy of the system.properties file. You do not need to reset the port afterwards.